scripts/build-docker.sh

```diff
@@ -4,10 +4,14 @@
 # by setting up the environment, building the application, and organizing the output 
 # into a organization specific subdirectiory in the 'dist' directory.
 
+# Clean potential previous builds
+
+rm -rf .next/
+
 mkdir -p dist
 mkdir -p dist/$NEXT_PUBLIC_PLANNER_ORG_ID
 echo "Running yarn setup && yarn build for $NEXT_PUBLIC_PLANNER_ORG_ID"
-yarn setup $NEXT_PUBLIC_PLANNER_ORG_ID
+yarn setup
```

Comment:
Isn't orgId required here?

![2023-10-31 at 13 31 17](https://github.com/AtB-AS/planner-web/assets/14045515/4a97827d-84ae-4636-8f05-219192acd293)

Reply:
Is it set in the context of where this script is run possibly

Reply:
But `NEXT_PUBLIC_PLANNER_ORG_ID` is set as an environment variable while running that script. Setup in yarn doesn't actually take any argument:


```
    "setup": "generate-assets all $NEXT_PUBLIC_PLANNER_ORG_ID -o ./public/assets/ -ts -ts-o ./src/components/icon",

```